src/lib/appManagers/appStateManager.ts

```diff
@@ -17,9 +17,10 @@ import rootScope from '../rootScope';
 import sessionStorage from '../sessionStorage';
 import { logger } from '../logger';
 import { copy, setDeepProperty, validateInitObject } from '../../helpers/object';
-import { getHeavyAnimationPromise } from '../../hooks/useHeavyAnimationCheck';
 import App from '../../config/app';
 import DEBUG, { MOUNT_CLASS_TO } from '../../config/debug';
+import AppStorage from '../storage';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MikeOwino/tweb/pullRequest?prid=7329411)
```suggestion
import AppStorage from "../storage";
```